### PR TITLE
WDP181201-10 and WDP181201-38 combined

### PR DESCRIPTION
**Pull request by lkulbacki**
Added animation effects to WDP181201-10 (https://github.com/OskarLebuda/wdp-1812-01/pull/4)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,577 +10,612 @@
...
-    <div class="top-bar">
-      <div class="container">
-        <div class="row">
-          <div class="col text-left">
-            <ul>
-              <li><a href="#">USD <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">English <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">Help <i class="fas fa-caret-down"></i></a></li>
-            </ul>
+<header>
```
Comment (pawel-schmidt) [inline src/index.html]: Next time please try to put code format in a separate branch. At the moment it is hard to find out which lines you have modified.

File: src/sass/components/_header.scss [hunk 2]
```diff
@@ -176,6 +184,56 @@ header {
       }
     }
 
+    .submenu {
+      @extend %animate;
+      display: flex;
+      visibility: hidden;
+      opacity: 0;
+      background-color: #363636;
```
Comment (pawel-schmidt) [inline src/sass/components/_header.scss]: I would rather use the SCSS variable. We are using this color in another place on the page, so it's highly probable we will be changing both colors at the same time.

File: src/index.html (hunk 1 above)
Comment (lkulbacki) [reply src/index.html]: Separate branch or separate commit? 😕 I think just formatting code is way to small change to make a separate branch to it

File: src/sass/components/_header.scss (hunk 2 above)
Comment (lkulbacki) [reply src/sass/components/_header.scss]: I have changed this particular colour code to variable. In overall, project does not rely on colours based on variables. Might be a good idea to unify this approach at some point.

File: src/index.html (hunk 1 above)
Comment (pawel-schmidt) [reply src/index.html]: Separate branch. Let's say we have task `Fix typo in HTML`. You are going to change one of 10k lines in HTML and formatted whole code. Git will say, you have changed ~10k lines in PR entitled `Fixed typo`.
Of course, code added/modified by you should be properly formatted.

File: src/sass/components/_header.scss (hunk 2 above)
Comment (pawel-schmidt) [reply src/sass/components/_header.scss]: Yup, unification is almost always a good idea ;-)

**Comment by pawel-schmidt** [comment]
Please resolve merge conflicts.

**Comment by lkulbacki** [comment]
Merge conflicts resolved

File: src/index.html (hunk 1 above)
Comment (lkulbacki) [reply src/index.html]: Discussion finalised on Skype. Thanks for explanations!

**Pull request merged**